src/libraries/Common/src/System/Obsoletions.cs

```diff
@@ -39,5 +39,8 @@ internal static class Obsoletions
 
         internal const string BinaryFormatterMessage = "BinaryFormatter serialization is obsolete and should not be used. See https://aka.ms/binaryformatter for more information.";
         internal const string BinaryFormatterDiagId = "SYSLIB0011";
+
+        internal const string CodeBaseMessage = "CodeBase and EscapedCodeBase are only included for .NET Framework compatibility. Use Location instead.";
```

Comment:
```suggestion
        internal const string CodeBaseMessage = "Use Location instead.";
```

The message approved by API-review is just "Use Location instead.": https://github.com/dotnet/runtime/issues/31127#issuecomment-545084027 . 

None of the other messages say that the APIs are only included for .NET Framework compatibility, even though number of them could.